modules/git/git.go

```diff
@@ -71,6 +76,12 @@ func BinVersion() (string, error) {
 }
 
 func init() {
+	absPath, err := exec.LookPath("git")
```

Comment:
I think this is unnecessary and we should change many other places git command invokes and add an option on app.ini.

Reply:
Even if we later move to a config option will would just have to set GitExecutable to the config option.

Reply:
Maybe user have given it an absolute path?
```Go
if !filepath.IsAbs(GitExecutable) {
    absPath, err := exec.LookPath(GitExecutable)
    if err != nil {
		panic(fmt.Sprintf("Git not found: %v", err))
	}
	GitExecutable = absPath
}
```

Reply:
According to https://golang.org/pkg/os/exec/#LookPath
If `GitExecutatble` contains a slash, it would just try that path (could be a relative or absolute path).

So, I think `IsAbs` is not necessary.
Also, I think giving users to ability to specify a relative path has its merits.  Say, a user puts a custom `git` under a subdirectory of the root directory of `gitea`.

Reply:
I think you are right!